docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -22,6 +22,11 @@
             <td style="word-wrap: break-word;">"0"</td>
             <td>With this configuration option, users can specify a port, a range of ports or a list of ports for the Application Master (and JobManager) RPC port. By default we recommend using the default value (0) to let the operating system choose an appropriate port. In particular when multiple AMs are running on the same physical host, fixed port assignments prevent the AM from starting. For example when running Flink on YARN on an environment with a restrictive firewall, this option allows specifying a range of allowed ports.</td>
         </tr>
+        <tr>
+            <td><h5>yarn.application.priority</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>The priority which Flink used to submit YARN application. The priority is non negative, the bigger the number, the higher the priority. By default, we take -1. When the priority is negative, we use default yarn queue priority.</td>
```

Comment:
maybe it is nice to hyperlink with YARN's priority scheduling doc [here](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html), 

Just out of top of my head.
```
A non-negative integer indicating the priority for submitting a Flink YARN application. 
It will only take effect if YARN priority scheduling setting is enabled. 
Larger integer corresponds with higher the priority. 
If priority is set to '-1', Flink will unset yarn priority setting and use cluster default priority, see [YARN Capacity Scheduling Doc](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html) for more info.
```

Reply:
It is very useful, I have changed the option description according to this.